mlflow/store/db/utils.py

```diff
@@ -60,7 +61,8 @@ def _get_package_dir():
 
 
 def _all_tables_exist(engine):
-    expected_tables = {
+    return set(sqlalchemy.inspect(engine).get_table_names()) == {
+        "alembic_version",
```

Comment:
Can we use

```
expected_tables.issubset(sqlalchemy.inspect(engine).get_table_names())
```
?

So that if alembic added more meta table in future, we don't need to update the test again.

Reply:
> if alembic added more internal table in future

I don't think this will happen.

Reply:
we could filter out tables starting with `albembic`.

Reply:
Sounds good. :)

Reply:
Updated!